diff --git a/src/md5_default.sh b/src/md5_default.sh
--- a/src/md5_default.sh
+++ b/src/md5_default.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!@usrBinEnv@ bash
 #
 # Copyright 2022 The Bazel Authors. All rights reserved.
 #
diff --git a/src/md5_darwin_freebsd.sh b/src/md5_darwin_freebsd.sh
--- a/src/md5_darwin_freebsd.sh
+++ b/src/md5_darwin_freebsd.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!@usrBinEnv@ bash
 #
 # Copyright 2022 The Bazel Authors. All rights reserved.
 #
diff --git a/src/md5_openbsd.sh b/src/md5_openbsd.sh
--- a/src/md5_openbsd.sh
+++ b/src/md5_openbsd.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!@usrBinEnv@ bash
 #
 # Copyright 2022 The Bazel Authors. All rights reserved.
 #
